src/pubsub.ts

```diff
@@ -300,6 +318,7 @@ export class PubSub {
     if (this.isOpen) {
       this.isOpen = false;
       this.closeAllClients_()
+        .then(() => this.schemaClient?.close())
```

Comment:
Just wondering why the `schemaClient` should be closed separately and not as a part of `closeAllClients_()`.

Reply:
Ah, yeah, this is me breaking with tradition from the old code. Previously, the library would store a bunch of raw gax clients by service name, and then call into them directly. The new way that I'm using for schema is to create a standard gapic object and store that for usage on schema calls. So it's not part of the "all clients" collection.

Eventually I'd like to eliminate all of the old "all clients" bits and make things use gapic underneath. But for now I'm keeping it separate to make it obvious.